### PR TITLE
[FEAT] Create onboarding sequence of info graphics pages

### LINKED ISSUE #11: Build QTime onboarding UI pages (UI1.5-1.7)
Create the onboarding pages for the QTime UI1.5-1.7.

## Definition of done
* The onboarding pages for QTime are complete.

### PR DESCRIPTION
**Pull request by benjaminkostiuk**
## Overview

Implements the onboarding infographics for new users + help section.

Closes #11.

This change is a

- [ ] Bug fix
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that will change existing functionality)
- [ ] Documentation update

## Preview
Animation

Animation

## Description

 
- Add new onboarding assets
- Create new `OnboardingScreen` component that has the carousel of onboarding info pages. This is used during onboarding and also when the user presses help.
- Bunch of comment fixes and re-naming stuff.
- Update profile screen to display a modal with the onboarding pages on it when they press on help.
- Refactor referral page to transition into the onboarding pages.
- Fix referral page tests
- Add tests for the onboarding component. For the life of me I can't get the scroll to work in testing so consider it manually tested for the complete button. If I have time I'll fix it but for now 4+ hours in I can't get it to work.

## Motivation/Links
This lets us show users an onboarding experience with a series of info pages they can go through, and come back to anytime.

### CODE REVIEW

File: src/navigators/SignUpStackNavigator.tsx [hunk 1]
```diff
@@ -47,6 +45,9 @@ export const SignUpStackNavigator = () => (
       component={TermsOfServiceScreen}
     />
     <Stack.Screen name={ROUTES.REFERRAL} component={ReferralScreen} />
-    <Stack.Screen name={ROUTES.ONBOARDING} component={OnboardingScreen} />
+    <Stack.Screen
+      name={ROUTES.ONBOARDING}
+      component={OnboardingScreenForNavigator}
```
Comment (benjaminkostiuk) [inline src/navigators/SignUpStackNavigator.tsx:50]: This is to get around having to pass a prop to the component. It's much easier this way, even if it's a little bit of a workaround.

File: src/screens/OnboardingScreen.tsx [hunk 2]
```diff
@@ -1,5 +1,269 @@
...
+              <Feather name="check" size={20} color="#ffffff" />
+            </Button>
+          )}
+        </Animated.View>
+      </View>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
```
Comment (benjaminkostiuk) [inline src/screens/OnboardingScreen.tsx:210]: We need to look into a better way of managing styles used in components that having this big long list for each of them.

Comment (dhruv423) [reply src/screens/OnboardingScreen.tsx:210]: I think for our time, its probably for the better

Comment (benjaminkostiuk) [reply src/screens/OnboardingScreen.tsx:210]: Yeah for now it's not a big deal, but maybe down the line it's something we could refactor.

**Review by dhruv423 — APPROVED** [review]

**Pull request merged**